rllib/algorithms/ppo/ppo.py

```diff
@@ -70,37 +70,39 @@ class PPOConfig(AlgorithmConfig):
     .. testcode::
 
         from ray.rllib.algorithms.ppo import PPOConfig
+
         config = PPOConfig()
-        config = config.training(gamma=0.9, lr=0.01, kl_coeff=0.3,
-            train_batch_size=128)
-        config = config.resources(num_gpus=0)
-        config = config.env_runners(num_env_runners=1)
+        config.environment("CartPole-v1")
+        config.training(
+            gamma=0.9, lr=0.01, kl_coeff=0.3, train_batch_size_per_learner=256
+        )
+        config.resources(num_gpus=0)
```

Comment:
This is still ambiguous. We have `num_gpus` in `resources`, `num_gpus_per_learner` in `learners` and `num_gpus_per_env_runner`. What is this `gpu` for? Is it a gpu for the driver? Is it the total number of gpus available to env runners and learners?